scsi_bsg_util.c

```diff
@@ -280,3 +315,63 @@ int send_bsg_scsi_trs(int fd, struct ufs_bsg_request *request_buff,
 		reply_buff->reply_payload_rcv_len);
 	return ret;
 }
+
+/*
+ * submit_sec_cdb - Used to send SECURITY PROTOCOL OUT/IN CDB
+ * @fd: bsg driver file descriptor
+ * @spsp: SECURITY PROTOCOL SPECIFIC in CDB
+ * @secp: SECURITY PROTOCOL in CDB
+ * @buf: pointer to the SCSI cmd data buffer
+ * @len: SCSI data length
+ * @send: The cmd direction, True or 1 for send, False or 0 for read
+ */
+int submit_sec_cdb(int fd, __u16 spsp, __u8 secp, char *buf,
```

Comment:
security protocol in / out is rpmb only - maybe this function belongs to rpmb.c ?